plasmapy/dispersion/numerical/tests/test_hollweg_.py

```diff
@@ -116,6 +116,81 @@ def test_warning(self, kwargs, _warning):
         with pytest.warns(_warning):
             hollweg(**kwargs)
 
+    @pytest.mark.parametrize(
+        "kwargs, expected",
+        [
+            (  # array input for k, single value for theta
+                {
+                    **_kwargs_single_valued,
+                    "k": np.logspace(-7, -2, 2) * u.rad / u.m,
```

Comment:
Why are you using negative values for `k` here?  Since we're using `k` and `theta` as input arguments, I would think we would require `k > 0`.

Reply:
It's `np.logspace` so it should range from [10^-7,10^-2] and be positive right?

Reply:
ah...I see...my brain automatically read `np.linspace`